contribute-docs/style-guide/ui-writing.md

```diff
@@ -0,0 +1,324 @@
+| 11.    | **time filter** <br /> Specifies the data you want to display within a specified time range.   |
+| 12.    | **Refresh** <br /> Refreshes the data that is displayed.  |
+
+![Kibana with labels](./images/namingKibana02.png)
+
+| Number | Definition |
+|--------|------------|
+|  1.    | **fields list** <br /> The list of data fields that are available based on your index pattern and the time filter. <br /> When referring to individual lists, use: <br /> &bull; **Available fields** list <br /> &bull; **Empty fields** list <br /> &bull; **Meta fields** list <br /> When referring to the action of adding fields, use `drag x to the x…`. For example, _from the **Available fields** list, drag **agent.keyword** to the workspace_.  |
+|  2.    | **Filter by type dropdown** <br /> Specifies the filters you want to apply to the fields list.  |
+|  3.    | **Index pattern dropdown** <br /> Opens the **Index pattern** dropdown, where you can select a different index pattern. <br /> In general, refer to all dropdown menus in Kibana as a _dropdown_.  |
+|  4.    | **action menu (…)** <br /> Opens a menu that allows you to add fields to the index pattern, and manage index pattern fields. <br /> For example, _open the action menu (…) next to the Index pattern dropdown_.  |
+|  5.    | **Visualization type dropdown** <br /> Opens the Visualization type dropdown, where you can select a different visualization.  |
+|  6.    | **editor toolbar menus** <br /> Opens menus that allows you to customize the visualization. <br /> For example, _in the editor toolbar, open the **Legend** menu_.  |
+|  7.    | **workspace** <br /> The space where you drag fields and create visualizations. <br /> To describe how you add data fields to the workspace, use _drag_ instead of _drag and drop_. For example, _from the **Available fields** list, drag bytes to the workspace_. <br /> To describe how you zoom in on a data set, never use _brush_. Instead, use _click and drag your cursor_. For example, _to zoom in on the data you want to view, click and drag your cursor across the bars_.  |
+|  8.    | **Chart type menu** <br /> Opens the **Chart type** menu, where you can click a different chart type for the layer. Available only for area, line, and bar charts.  |
+|  9.    | **layer pane** <br /> The layer configuration options, such as fields, functions, formula, colors, and more.  |
+| 10.    | **Index pattern dropdown** <br /> Opens the **Index pattern dropdown**, where you can select a different index pattern for the layer.  |
+| 11.    | **Horizontal axis, Vertical axis, Break down by, etc.** <br /> Refers to the field groups. <br /> To be generic, use _drag fields to the layer pane_.  |
+| 12.    | **Suggestions** <br /> The alternative visualizations that **Lens** creates for your data.  |
+
+
+### Cloud, labeled
+
+![Cloud with labels](./images/cloudLabeled.png)
```

Comment:
I think this image is also quite out-dated (doesn't show "hosted deployments", "serverless deployments" sections) so should likely be commented out until it can be refreshed.

> ![Cloud with labels](./images/cloudLabeled.png)